samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -83,7 +83,8 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
     () => ({
       callReadinessOptedIn: callReadinessOptedIn,
       onPermissionsTroubleshootingClick,
-      onNetworkingTroubleShootingClick
+      onNetworkingTroubleShootingClick,
+      unsupportedBrowserOptedIn: false
```

Comment:
why not opt in on calling sample also?

Reply:
Call readiness if off because the feature isn't complete and its a bad join experience right now, but I think this should be on by default -- or another way to ask the question; when would be the right time to turn it on?

Reply:
I will update this to reflect that we want it on now.